stress_test_model_eq.R

```diff
@@ -442,11 +442,14 @@ for (i in seq(1, nrow(transition_scenarios))) {
       )
   }
 
+  rows_equity <- nrow(equity_annual_profits)
```

Comment:
We lose a very higher number (> 10000) rows here, Let us hope this is due to either inconsistencies to cb workflow or due to the fact that import of pacta results is currently not really functional.

Reply:
I will make the eq workflow consistent with the cb workflow as part of this https://dev.azure.com/2DegreesInvesting/2DegreesInvesting/_workitems/edit/2182 (including adding the Credit Risk calculations). We will know more then, meaining I will go into why this may be happening